### DISCUSSION TITLE
【求助】可以发涩图吗

### DISCUSSION THREAD

**Discussion by hfwang132** [share121/inter-knot#1969] [其他]
![2024-04-19 04-37-46](https://github.com/user-attachments/assets/964fb8bf-c1a7-4c74-a428-249092d02b9f)

RT

**Comment by capacitor1** [comment]
![开拓者 吃瓜ing](https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png)

涩。

**Reply by wushang114** [reply]
为什么我的绳网打不开了？

**Comment by 1451374815** [comment]
尺度别太大

**Comment by yyxzz808** [comment]
不可以哦这位绳匠

**Comment by bbdqz** [comment]
布什,戈门

**Comment by hfwang132** [comment]
我指的是r15但不r18的，比如这张，嘿嘿

**Reply by bbdqz** [reply]
申么?发给我立刻!

**Comment by JohnDoe-A** [comment]
这位绳匠，本世界的绳网并非法外之地！

**Comment by Buer-Nahida** [comment]
非R18(G)就行

**Comment by qiang-di** [comment]
逆天

**Comment by Tiyi189** [comment]
ヽ(*ﾟｰﾟ*)ﾉ

**Comment by Leo2126** [comment]
敢发？分分钟给你墙了（bushi）

**Comment by Yinko-Livi** [comment]
6

**Comment by likeseele** [comment]
这张我喜欢，韩了

**Comment by Rainy-poi** [comment]
现在看不了了

**Comment by Bertdada** [comment]
我的绳网只能看到模糊的啊啊啊啊啊啊啊啊

**Reply by HifumiMizuhara** [reply]
在GitHub里可以看到清晰的

**Comment by chen92525456** [comment]
请务必发给我
![99c9d000ec7bb80edfb2840ddcf3156](https://github.com/user-attachments/assets/c08057d6-46fc-469e-8254-35b26db4037e)

**Reply by March3th** [reply]
花火大人表示这很欢愉
![0e0c0c9c-375d-43df-85de-c18743fbc906](https://github.com/user-attachments/assets/754166f7-9155-4019-ab0e-c9bdb0877e9c)

**Comment by haifanhen6** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by chen92525456** [comment]
6



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月15日 12:26 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 【求助】可以发涩图吗 (Discussion #1969) |

举报了

**Comment by Hejin1212** [comment]
快发！！<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Genshin-Neuvillette** [comment]
芙芙的可以

**Comment by Kamisato100** [comment]
![CharaStudio-2023-11-01-01-42-39-Render](https://github.com/user-attachments/assets/8ed195ce-47e9-41aa-8f2a-65029a300afd)

**Comment by ywhuuuu** [comment]
想发色图，直接发给我我看怎么个事。

**Comment by maopao111** [comment]
别太过分就行了，r16我爱看<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by shengwangyonghu** [comment]
绳网不是不法之地！！！（但这个是doge)
